proctord/storage/postgres/client_test.go

```diff
@@ -44,3 +44,35 @@ func TestNamedExec(t *testing.T) {
 	_, err = postgresClient.db.Exec("DELETE FROM jobs_execution_audit_log WHERE job_name='test-job-name'")
 	assert.NoError(t, err)
 }
+
+func TestSelect(t *testing.T) {
+	dataSourceName := fmt.Sprintf("dbname=%s user=%s password=%s host=%s sslmode=disable", config.PostgresDatabase(), config.PostgresUser(), config.PostgresPassword(), config.PostgresHost())
+
+	db, err := sqlx.Connect("postgres", dataSourceName)
+	assert.NoError(t, err)
+
+	postgresClient := &client{db: db}
+	defer postgresClient.db.Close()
+	jobName := "test-job-name"
+
+	jobsExecutionAuditLog := &JobsExecutionAuditLog{
+		JobName:                  jobName,
+		ImageName:                "test-image-name",
+		JobSubmittedForExecution: "test-submission-name",
+		JobArgs:                  "test-job-args",
+		JobSubmissionStatus:      "test-job-status",
+		JobExecutionStatus:       "test-job-execution-status",
+	}
+
+	err = postgresClient.NamedExec("INSERT INTO jobs_execution_audit_log (job_name, image_name, job_submitted_for_execution, job_args, job_submission_status, job_execution_status) VALUES (:job_name, :image_name, :job_submitted_for_execution, :job_args, :job_submission_status, :job_execution_status)", jobsExecutionAuditLog)
```

Comment:
@prashant3863 Can we add a test for select when no rows exist in the db with given job name? That'll satisfy the job not found condition.